install/sct_launcher

```diff
@@ -1,130 +0,0 @@
-#!/bin/bash
-#
-# Launcher for SCT
-#
-# Do not modify this file
-# Your modify SCT can be stored in your ${HOME}/.sctrc file
-# e.g.
-# To limit the number of thread used by some of the tools to N
-#   echo "ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS = N" >> $HOME/.sctrc
-#
-#########################################################################################
-# Copyright (c) 2016 Polytechnique Montreal <www.neuro.polymtl.ca>
-# Author: PO Quirion
-# License: see the file LICENSE.TXT
-#########################################################################################
-
-
-function install_conda {
-  echo noting yet
-}
-
-
-function set_system_dependent_var {
-  # Retrieve OS and number of CPU cores
-  if uname -a | grep -i  darwin > /dev/null 2>&1; then
-      # Do something under Mac OS X platform
-    SCT_OS=osx
-    logicalCpuCount=$(sysctl -n hw.logicalcpu_max)
-  elif uname -a | grep -i  linux > /dev/null 2>&1; then
-    SCT_OS=linux
-    logicalCpuCount=$(lscpu -p | egrep -v '^#' | wc -l)
-  else
-    echo YOUR OPERATING SYSTEM IS NOT SUPPORTED YET
-  fi
-}
-
-
-function addtopath {
-  # will not add the path again if it is already there
-  # usage: addtopath "/a/directory/to/be/added" PATH_VARIABLE
-  if [[ ":${!2}:" != *":$1:"* ]]; then
-    if [[ ! -z ${!2} ]]; then
-        bidon1="${1}:${!2}"
-    else
-        bidon1="${1}"
-    fi
-    eval "${2}=$bidon1"
-  fi
-}
-
-
-if [ -z "${SCT_LOADED_ONCE}" ]; then
-  # Do not needs to be reloaded on subsequent system calls
-  export SCT_LOADED_ONCE="loaded"
-  # exclude ~/.local/lib/python (linux) or ~/Library/Python (OSX) libraries
-  export PYTHONNOUSERSITE=1
-  # set at installation
-  set_system_dependent_var
-
-  SCT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-  export SCT_DIR=${SCT_DIR%/bin}
-  export SCT_PYTHON=${SCT_DIR}/python
-  export SCT_VERSION=$(cat $SCT_DIR/version.txt)
-
-  export PATH
-  export PYTHONPATH
-  export ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS=${logicalCpuCount}
-
-  {DYNAMIC_INSTALL_TEMPLATE}
-
-  if [ -e ${HOME}/.sctrc ]; then
-  # Overwrite parameter with local config
-    . ${HOME}/.sctrc
-  fi
-
-
-  # Will use system python if conda or virtual env are not used
-  echo ""
-  . ${SCT_PYTHON}/bin/activate ${SCT_PYTHON}
-  actication=$?
-  if [[ ${actication} != 0 ]]; then
-    echo [WARNING] using  system python, ${SCT_PYTHON}/bin/activate not found
-  fi
-
-  addtopath "${SCT_DIR}/bin" PATH
-
-  # display SCT version
-  echo "--"
-  if [ -e ${SCT_DIR}/.git ]; then
-    # retrieve commit
-    SCT_COMMIT=`git --git-dir=${SCT_DIR}/.git rev-parse HEAD`
-    # retrieve branch
-    SCT_BRANCH=`git --git-dir=${SCT_DIR}/.git branch | grep \* | awk '{print $2}'`
-    echo "Spinal Cord Toolbox ($SCT_BRANCH/$SCT_COMMIT)"
```

Comment:
Any reason you removed this header upon launching scripts? It was cute and especially useful when users contact us with bugs (it helps to quickly find what version they are running)

Reply:
The whole launcher was removed, instead a Python launcher is used; the revision can still be found by running `sct_check_dependencies` or in Python with `spinalcordtoolbox.__version__`

Reply:
right, but this doesn't solve finding what version people run when they copy/paste their errors (it involves additional backandforth, asking to run command X to get the version...).

also, when running pipelines, i (and probably others) find it quite useful to know where new commands start by displaying "Spinal Cord Toolbox version XXX"

Lastly, when retrieving old logs from pipelines, knowing what SCT version was run can be useful.

any good reason for removing it?